lib/usagereporter/teleport/types.go

```diff
@@ -524,6 +538,12 @@ func ConvertUsageEvent(event *usageeventsv1.UsageEventOneOf, userMD UserMetadata
 		}
 
 		return ret, nil
+	case *usageeventsv1.UsageEventOneOf_UiCallToActionClickEvent:
+		return &UICallToActionClickEvent{
+			UserName: userMD.Username,
+			Cta:      prehogv1a.CTA(e.UiCallToActionClickEvent.Cta),
```

Comment:
If this is what we're doing then we need a comment near the usageevents definition of CTA to remind us to keep it in sync with the one from prehogv1alpha.

Reply:
Added some comments on `usageevents.proto`. Thanks.